numpy/core/tests/test_ufunc.py

```diff
@@ -36,6 +36,10 @@ def test_sig_dtype(self):
         assert_raises(RuntimeError, np.add, 1, 2, signature='ii->i',
                       dtype=int)
 
+    def test_extobj_refcount(self):
+        # Should not segfault with USE_DEBUG.
+        assert_raises(TypeError, np.add, 1, 2, extobj=[4096], parrot=True)
```

Comment:
`invalid_keyword_arg` would be a little clearer than `parrot`

Reply:
Hey, I finally got a Monty-Python reference in!